annotation/src/main/java/org/cbioportal/genome_nexus/annotation/web/AnnotationController.java

```diff
@@ -114,6 +120,15 @@ public AnnotationController(VariantAnnotationService variantAnnotationService,
             postEnrichmentService.registerEnricher(isoformOverrideSource, enricher);
         }
 
+        // ignore any other value than "full" or "summary"
+        if (cancerHotspots != null &&
+            (cancerHotspots.equalsIgnoreCase("full") || cancerHotspots.equalsIgnoreCase("summary")))
```

Comment:
Agreed. It is generally a good idea to have final strings or enums instead of hardcoded strings.